src/DivAndCon.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class DivAndCon {
+
+    public static void main(String[] args) {
+
+        int[] rndArray=fillRandomly();
+
+        System.out.println("Your Array is: ");
+        for (int i = 0; i < rndArray.length; i++){
+            System.out.print("[" + rndArray[i] + "]");
+        }
+
+       findMax(rndArray, rndArray);
+
+    }
+
+    public static int[] fillRandomly (){
+
+        Scanner scan = new Scanner(System.in);
+        Random rnd = new Random();
+
+        System.out.println("Define the length of your Array.");
+        int arrayLength = scan.nextInt();
+
+        int[] emptyArray = new int[arrayLength];
+
+        for (int i = 0; i < emptyArray.length; i++){
+            emptyArray[i] = rnd.nextInt(100+1);
+
+        }
+        return emptyArray;
+    }
+
+    public static int findMax(int[] yourArray, int[] initialArray){
+        int[] tempArray = initialArray;
+        int max = tempArray[0];
+
+       if (yourArray.length == 1){
+
+           for (int i = 0; i < tempArray.length; i++){
```

Comment:
🤔 
macht das sinn, dass wir hier über ein array iterieren dass sowieso nur ein element hat? wenn wir doch schon wissen dass in max unser erstes element ist und das auch unser einziges ist, dann können wir hier einfach das zurückgeben. d.h. Zeilen 43 bis 51 machn für mich wenig Sinn, vlt kannst du  mir erklären was du dir hierbei gedacht hast?

Reply:
Also dabei gehts hauptsächlich um den Konsolen Output. Der temp/initial Array ist defacto nichts anderes als ein Speicherort um den Ursprungsarray aufrecht zu erhalten und die temp Instanz habe ich implementiert um anhand dieser das Max herauszufinden. So kann ich dann den einstelligen Array mit dem Max vergleichen und diesen dann printen sofern er dem Maximum entspricht. Kann auch sein, dass das totaler Schwachsinn ist. Aber es funzt ;_;